src/Liquidator.sol

```diff
@@ -136,26 +163,32 @@ contract Liquidator is Ownable {
             return (0, false);
         }
 
-        price = _calcPriceOfVault(auctionInformation[vault].startTime, auctionInformation[vault].openDebt);
+        uint256 timePassed;
+        unchecked {
+            timePassed = block.timestamp - auctionInformation[vault].startTime;
+        }
+        if (timePassed > auctionCutoffTime) {
+            return (0, false);
+        }
+
+        price = _calcPriceOfVault(timePassed, auctionInformation[vault].openDebt);
     }
 
     /**
      * @notice Function returns the current auction price given time passed and the openDebt.
-     * @param startTime The timestamp when the auction started.
+     * @param timePassed delta between current time and auction start time.
      * @param openDebt The open debt taken by `originalOwner`.
      * @return price The total price for which the vault can be purchased.
      * @dev We use a dutch auction: price constantly decreases and the first bidder buys the vault
      * And immediately ends the auction.
      */
-    function _calcPriceOfVault(uint256 startTime, uint256 openDebt) internal view returns (uint256 price) {
-        uint256 auctionTime = block.timestamp - startTime; //Can be unchecked
-
-        if (auctionTime > maxAuctionTime) {
-            //ヽ༼ຈʖ̯ຈ༽ﾉ
-            price = 0;
-        } else {
-            price = uint256(openDebt) * startPriceMultiplier * (maxAuctionTime - auctionTime) / maxAuctionTime / 100;
+    function _calcPriceOfVault(uint256 timePassed, uint256 openDebt) internal view returns (uint256 price) {
+        uint256 auctionTime;
+        unchecked {
+            auctionTime = timePassed * 1_000_000_000_000_000_000;
```

Comment:
Make it 1e18 easier reading

Reply:
It is lower gas

Reply:
1e18 is the same in gas usage, but clearer to read